### DISCUSSION TITLE
How to upgrade vscode version?

### DISCUSSION THREAD

**Discussion by cheval1201** [coder/code-server#4372] [Q&A] [answered]
- `code-server` version: 3.12.0
- `vscode` version: 1.39.2
- `iPadOS` version: 15.0.2
- chrome browser

## Description
I am trying to install vscode extensions(vim, python), but I get errors saying current version of VSCode is not compatible with the extensions. 

Installing the extensions using `code-server` binary from the host server with the `—install-extension` option works, but won’t properly load when I open VSCode on remote.

**Comment by code-asher** [comment, marked as answer]
For now the only way to upgrade VS Code is to wait for a new release of code-server.  Our next release will have 1.61.1!

You might have better luck with installing the extensions by downloading them from Open VSX; the ones hosted in our marketplace can be finicky.
